Library/Formula/gegl.rb

```diff
@@ -64,4 +59,23 @@ def install
     system "./configure", *argv
     system "make", "install"
   end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <gegl.h>
+      gint main(gint argc, gchar **argv) {
+        gegl_init(&argc, &argv);
+        GeglNode *gegl = gegl_node_new ();
+        gegl_exit();
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-I#{include}/gegl-0.3", "-L#{lib}", "-lgegl-0.3",
+           "-I#{Formula["babl"].opt_include}/babl-0.1",
+           "-I#{Formula["glib"].opt_include}/glib-2.0",
+           "-I#{Formula["glib"].opt_lib}/glib-2.0/include",
+           "-L#{Formula["glib"].opt_lib}", "-lgobject-2.0", "-lglib-2.0",
+           testpath/"test.c", "-o", testpath/"test"
+    system "test", 0
```

Comment:
Ohhhhhhh, that's why I needed to add the 0.  I thought it was just something about the test code I didn't understand (I need someone to write C tests for me pls).